src/rnpkeys/rnpkeys.c

```diff
@@ -102,6 +105,8 @@ struct option options[] = {
   {"results", required_argument, NULL, OPT_RESULTS},
   {"cipher", required_argument, NULL, OPT_CIPHER},
   {"expert", no_argument, NULL, OPT_EXPERT},
+  {"output", required_argument, NULL, OPT_OUTPUT},
+  {"force", no_argument, NULL, OPT_FORCE},
```

Comment:
So, do you really want to call option ``--overwrite`` knowing that it won't be used only for overwriting?

Reply:
Basically, right now it's used for overwriting only, that's why discussion came up. Later on other options may be added, once corresponding functionality is implemented.

Reply:
Ok, I'll implement ``--overwrite`` once is needed then

Reply:
My question is - do you agree to reuse this option not only for overwriting?

Reply:
Not sure if I get question correctly. My opinion is that for file overwriting we should use `--overwrite`, as in `rnp`. For other things, like GnuPG does, `--yes` could be used.

Reply:
Yup, I want "other things". Cool, so instead of ``--yes`` we will call the option ``--force``. I hope this closes whole story.

Reply:
I tend to agree that `--force` is better than `--yes` because "force" ensures that the intent of the action is delivered. With "Yes", a question doesn't always have to be positive (imagine a question like "Do you not want to delete this file? Yes.").

Reply:
@ronaldtse I think the main issue that @ni4 and myself had with this, is not related to `--force` vs `--yes`. It was more that this PR added a `CFG_FORCE` option which is used here to enable overwriting files, despite the fact that we already have a `CFG_OVERWRITE` option used in the other utility for this purpose.

Reply:
Sorry I got confused. Indeed we should keep using the `CFG_OVERWRITE` option rather than `CFG_FORCE`.

However, `--force` seemed more reasonable than `--overwrite` in this context, or actually other OVERWRITE contexts.

What would you think if we use `force` instead of `overwrite` in those areas?

Reply:
I think we need (at least?) two options - one is something like "yes for all the questions" another one is "I know what I'm doing, so I want to force RNP to do it even if it destroys my keyrings"

We should use first of those two options instead of ``--overwrite`` (in the end specifying this options means - answer "yes" if rnp asks if I want to overwrite a file).

Reply:
It sounds like you both are saying that we should have a broad `--force` option, and remove the specific `--overwrite` option?

My view is that we should try to have both broad and granular options. So IMO:
* We should have an option like `--force`, whatever the name. This would enable overwriting and every other override.
* We should have more specific options like `--overwrite` too.

If I'm writing a shell script that uses rnp, I should be able to say "yes, you can overwrite files, but I don't want you to do anything else unusual". A broad option like `--force` will inevitably end up being used to indicate things like "yes I want to use this iteration count, even though it seems low" or "yes I want to use this hash alg, even though it's not in the prefs". With scripts, I would rather set very specific options and avoid broad things like `--force` that may have unintended consequences (plus the overrides enabled by `--force` will likely expand in future versions, so it's not a great option for scripts).

Anyways, that's my .02, do what you wish.

Reply:
I'm definitely not saying we should not have granular options. I'm saying we need ``--force`` and we should have ``--yes`` option. I don't see why we need ``--overwrite`` option, but if it's needed then well... let's have it.

>  broad option like --force will inevitably end up being used 

That's an opinion not a fact. It's not "will" but "may" and if it is then someone uses software wrongly.

> With scripts, I would rather set very specific options
Yes, I agree - you shouldn't use ``--force`` in scripts. You should use it with care.

Reply:
Let's continue on https://github.com/riboseinc/rnp/issues/560.